sylvia/Cargo.toml

```diff
@@ -24,12 +31,17 @@ serde = { version = "1.0", default-features = false, features = ["derive"] }
 serde-cw-value = "0.7.0"
 serde-json-wasm = "1.0.0"
 konst = "0.3"
-cw-multi-test = { version = "0.18.0", optional = true }
+cw-multi-test = { version = "0.18.0", optional = true, features = [
+    "cosmwasm_1_2",
```

Comment:
You cannot enable "cosmwasm_1_2" in mt and not enable it in cosmwasm-std. You are asking yourself for problems. You need to have a `cosmwasm_1_2` feature flag, which enables those two. Then everything that needs those flags to work, should be behind it (we can leave setting salt itself, but ignore it when instantiating without this flag).